contracts/Compass.vy

```diff
@@ -43,6 +55,11 @@ struct TokenSendArgs:
     receiver: DynArray[address, MAX_BATCH]
     amount: DynArray[uint256, MAX_BATCH]
 
+struct FeeArgs:
```

Comment:
@webelf101 This is still missing relayer_fee value (which will be higher than the `gas_estimate`)

Reply:
So, basically:

- For `update_valset` and `submit_batch`, relayers will be reimbursed the amount of `gas_estimate` from the security funds
- For `submit_logic_call`, relayers will be reimbursed with the value of `relayer_fee`, which will be `gas_estimate * 1.x` In addition, `community_fee` and `security_fee` amounts will have to be moved to their respective accounts. And all of this (`relayer_fee` + `security_fee` + `community_fee`) will come out of the funds belonging to `fee_payer_paloma_address`.

Reply:
Should we use the relayer fee instead of actual gas cost in SLC? @byte-bandit

Reply:
fixed. @byte-bandit